doc/Language/5to6-nutshell.pod6

```diff
@@ -1509,7 +1509,7 @@ Perl modules is C<PERL5LIB>.
 $ PERL5LIB="/some/module/lib" perl program.pl
 
 In Raku this is similar, one merely needs to change a number!  As you
-probably guessed, you just need to use X<C<PERL6LIB>|PERL6LIB>:
+probably guessed, you just need to use X<C<PERL6LIB>|p5 nutshell>:
```

Comment:
Um, this looks weird. This is going to be indexed as what's to the right of the |; it might be better to use `PERL6LIB,p5 nutshell`. That creates a new category, so use whatever category is already been used there.